packages/visual-editor/src/components/contentBlocks/MapboxStaticMap.tsx

```
@@ -8,10 +8,13 @@ import {
   YextField,
   msg,
   pt,
+  themeManagerCn,
+  Body,
 } from "@yext/visual-editor";
 import { ComponentConfig, Field, Fields } from "@measured/puck";
 import { StreamDocument } from "../../utils/applyTheme";
 import mapboxLogo from "../assets/mapbox-logo-black.svg";
+import { Map } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)